app/uk/gov/hmrc/plasticpackagingtax/registration/views/check_liability_details_answers_page.scala.html

```diff
@@ -33,11 +34,16 @@
   pageTitle: pageTitle,
   saveAndContinue: saveAndContinue
 )
-@(registration: Registration)(implicit request: Request[_], messages: Messages)
+@(registration: Registration)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@backLink = @{
```

Comment:
Just a suggestion, we can move this logic to the controller and pass back link as parameter to the page.

Reply:
This has been addressed.